cpp/src/parquet/encoding.cc

```diff
@@ -3086,7 +3257,17 @@ std::unique_ptr<Encoder> MakeEncoder(Type::type type_num, Encoding::type encodin
       default:
         throw ParquetException(
             "DELTA_BINARY_PACKED encoder only supports INT32 and INT64");
-        break;
+    }
+  } else if (encoding == Encoding::DELTA_LENGTH_BYTE_ARRAY) {
+    switch (type_num) {
+      case Type::BYTE_ARRAY:
+        return std::unique_ptr<Encoder>(
+            new DeltaLengthByteArrayEncoder<ByteArray>(descr, pool));
+      case Type::FIXED_LEN_BYTE_ARRAY:
+        return std::unique_ptr<Encoder>(
+            new DeltaLengthByteArrayEncoder<FLBAType>(descr, pool));
+      default:
+        throw ParquetException("DELTA_LENGTH_BYTE_ARRAY only supports BYTE_ARRAY");
```

Comment:
and `FIXED_LENGTH_BYTE_ARRAY`?

Reply:
I added the comment, but this doesn't support it yet.